src/scripting/events.cpp

```diff
@@ -115,7 +115,7 @@ void trigger_national_event(sys::state& state, dcon::national_event_id e, dcon::
 			n, dcon::nation_id{}, dcon::nation_id{},
 			sys::message_base_type::major_event
 		});
-	} else if(n == state.local_player_nation) {
```

Comment:
this and the one following was on purpose to prevent slowing down the update with notification sync. It isn't trivial